frontend/src/components/common/ModalAlert/ModalAlert.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
 import { Modal, Alert } from 'antd';
 
 export interface IModalAlertProps {
-  headTitle: string;
+  headTitle: string | ReactNode;
```

Comment:
```suggestion
  headTitle: ReactNode;
```
Doesn't this work? I mean, can't a string be considered a ReactNode already?